packages/amplify-cli/src/__tests__/commands/init.test.ts

```diff
@@ -20,14 +37,12 @@ jest.mock('which');
 (readdirSync as jest.Mock).mockReturnValue([]);
 (sync as jest.MockedFunction<typeof sync>).mockReturnValue('mock/path');
 (getPackageManager as jest.MockedFunction<typeof getPackageManager>).mockReturnValue(
-  new Promise((resolve) =>
-    resolve({
-      executable: 'yarn',
-      lockFile: 'mock.lock',
-      packageManager: 'yarn',
-      version: coerce('1.22.0') ?? undefined,
-    }),
-  ),
+  new Promise((resolve) => {
```

Comment:
nit: use `mockResolvedValue` without the `new Promise` wrapper

Reply:
updated